spec/takeaway_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'takeaway'
+
+describe Takeaway do
+  it 'lists available dishes and their prices' do
+    expect(subject.list_dishes).to eq("Hamburger: £1, Pizza: £2, Hotdog: £2, Tacos: £3, Nachos: £1")
+  end
+  describe '#add' do
+    it 'can choose dishes from the menu and add them to the order' do
+      subject.add("Pizza")
+      subject.add("Nachos", 1)
+      expect(subject.order).to eq ([{name: "Pizza", price: 2, quantity: 1}, {name: "Nachos", price: 1, quantity: 1}])
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [117/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)